changelogs/7.6.asciidoc

```diff
@@ -12,7 +12,7 @@ https://github.com/elastic/apm-server/compare/v7.5.1\...v7.6.0[View commits]
 
 [float]
 ==== Breaking Changes
-- Respect `apm-server.ilm.setup.overwrite` flag when running `setup --index-management` {pull}2984[2984].
+- Respect `apm-server.ilm.setup.overwrite` flag when running `setup --index-management` {pull}2959[2959].
```

Comment:
Actually this was a fix for a bug introduced and fixed during 7.5 development cycle, as part of the 7.5 breaking change: _Introduce dedicated ILM setup flags and ignore `setup.template.*` flags for ILM {pull}2764[2764], {pull}2877[2877]._.
Therefore I should have never added this changelog entry, especially not for `7.6`. Can you please completely remove it?

Great find!

Reply:
Will do